### PR TITLE
 Implemented cards in the workshopping page

### PR DESCRIPTION
**Pull request by rashikashaw**
The following cards are implemented in the workshopping page:

The idea name + tagline card
Description card
Idea features card
Target audience card
Cards are web responsive and adapt for mobile, tablets and desktop breakpoints.

### CODE REVIEW

File: apps/ideaspace/src/components/modules/WorkshoppingPage/CommentsForm/SingleComment.js [hunk 1]
```diff
@@ -13,11 +12,10 @@ const SingleCommentComponent = props => (
     <UserImage alt="user_image" src={`https://picsum.photos/70?random=${props.id}`} />
     <div className="textContent">
       <SingleCommentContent>
-        <h3>{props.author} - {props.author.role}</h3> {/* TODO: replace "idea owner" w/ a variable that contains the user's role pertaining to the idea (Iris is working on that) */}
+        <h3>{props.author}</h3>
         <div source={props.children} ><p>{props.children}</p></div>
       </SingleCommentContent>
       <SingleCommentButtons>
-        <LikeButton></LikeButton>
```
Comment (dbradham) [inline apps/ideaspace/src/components/modules/WorkshoppingPage/CommentsForm/SingleComment.js:20]: IS there any reason we are removing the like button here?

File: apps/ideaspace/src/components/modules/WorkshoppingPage/IdeaOverview/DescriptionCard/StyledComponents.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+export const StyledCard = styled(Card)`
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+  padding-left: 30px;
+  padding-right: 30px;
+  margin-bottom: 25px;
+`;
+
+export const Cardheading = styled.div`
```
Comment (dbradham) [inline apps/ideaspace/src/components/modules/WorkshoppingPage/IdeaOverview/DescriptionCard/StyledComponents.js:13]: could we change `Cardheading` into an `<h3>` to remove the nested `<h3>` elements in each of the `<Cardheading>` components?

File: apps/ideaspace/src/components/modules/WorkshoppingPage/IdeaOverview/TargetAudienceCard/StyledComponents.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  flex-direction: column;
+  align-items: start;
+  justify-content: center;
+  padding: 20px;
+  padding-left: 30px;
+  padding-right: 30px;
+  margin-top: 25px;
+`;
+
+export const Cardheading = styled.div`
```
Comment (dbradham) [inline apps/ideaspace/src/components/modules/WorkshoppingPage/IdeaOverview/TargetAudienceCard/StyledComponents.js:15]: It looks like a lot of these `Cardheading` styled components are repeated... Could we consolidate them into a single styled component?

File: apps/ideaspace/src/components/modules/WorkshoppingPage/WorkshoppingPage.js [hunk 4]
```diff
@@ -1,92 +1,57 @@
...
-      comments: [''],
-    }
-  );
-  // const location = useLocation();
-  // const pathname = location.pathname;
-  // const ideaId = pathname.slice(14, 19);
+} from './CommentsForm/StyledComments.js';
+import { useFetchIdea } from './useFetchIdea';
+
+const Wrapper = styled.div`
```
Comment (dbradham) [inline apps/ideaspace/src/components/modules/WorkshoppingPage/WorkshoppingPage.js:16]: Can we keep the styled components in a separate file?

**Comment by alicefate** [comment]
workshopping page merge all lines into one line.

**Pull request merged**